rootfs/scheduler/k8s.py

```diff
@@ -68,7 +68,80 @@
         "containers": [
           {
             "name": "$containername",
-            "image": "$image"
+            "image": "$image",
+            "env": [
+            {
+                "name":"DEIS_APP",
+                "value":"$id"
+            },
+            {
+                "name":"DEIS_RELEASE",
+                "value":"$appversion"
+            }
+            ]
+          }
+        ]
+      }
+    }
+  }
+}
+"""
+
+RCB_TEMPLATE = """\
+{
+  "kind": "ReplicationController",
+  "apiVersion": "$version",
+  "metadata": {
+    "name": "$name",
+    "labels": {
+      "app": "$id",
+      "heritage": "deis"
+    }
+  },
+  "spec": {
+    "replicas": $num,
+    "selector": {
+      "app": "$id",
+      "version": "$appversion",
+      "type": "$type",
+      "heritage": "deis"
+    },
+    "template": {
+      "metadata": {
+        "labels": {
+          "app": "$id",
+          "version": "$appversion",
+          "type": "$type",
+          "heritage": "deis"
+        }
+      },
+      "spec": {
+        "containers": [
+          {
+            "name": "$containername",
+            "image": "smothiki/slugrunner:4",
```

Comment:
looks like we have [slugrunner builds](https://travis-ci.org/deis/slugrunner) working now, so this should be [`quay.io/deisci/slugrunner`](https://quay.io/repository/deisci/slugrunner?tag=latest&tab=tags)